src/frontend/src/lib/stores/last-used-identities.store.ts

```diff
@@ -64,7 +66,14 @@ export const initLastUsedIdentitiesStore = (): LastUsedIdentitiesStore => {
         if (isNullish(identity.accounts)) {
           identity.accounts = {};
         }
-        identity.accounts[params.origin] = {
+        if (isNullish(identity.accounts[params.origin])) {
+          identity.accounts[params.origin] = {};
+        }
+        identity.accounts[params.origin][
+          isNullish(params.accountNumber)
+            ? "primary"
```

Comment:
Can you create a constant for this? This way it's more obvious that it's hardcoded and we can explain why.